src/witan/models/dem/ccm/core/projection_loop.clj

```diff
@@ -157,7 +157,7 @@
                         remove-deaths
                         apply-migration
                         join-popn-latest-yr)]
-        (println (format "Projecting for year %d..." (:loop-year inputs')))
+        (info (format "\nProjecting for year %d..." (:loop-year inputs')))
```

Comment:
I tend to prefer :refer :as timbre or log since that makes is clear it's imported from a certain namespace (nitpick though)